lib/organisms/gravity-form/gravity-form.directive.js

```diff
@@ -1,35 +1,22 @@
-(function () {
-
-    /**
-     * Created by Chris on 12/04/2016.
-     */
-
-    'use strict';
-
-
-    function lnOGravityForm() {
-        return {
-            restrict: 'A',
-            transclude: true,
-            templateUrl: 'lnPatterns/organisms/gravity-form/template.html',
-            scope: {},
-            controller: 'lnOGravityFormController',
-            controllerAs: '$ctrl',
-            bindToController: {
-                id: '@lnOGravityForm',
-                onSubmit: '&lnSubmit',
-                onError: '&lnError',
-                loadFromAPI: '=lnLoadFromApi'
-            }
+angular
+    .module('lnPatterns')
```

Comment:
I think this is good, because we can also include other angular modules inside ln-patternlab package in the future.